src/data/reports/glass-admin/GLASSDataMaintenanceDefaultRepository.ts

```diff
@@ -0,0 +1,492 @@
+import {
+    ATCOptions,
+    GLASSDataMaintenanceOptions,
+    GLASSDataMaintenanceRepository,
+} from "../../../domain/reports/glass-admin/repositories/GLASSDataMaintenanceRepository";
+import { DataStoreStorageClient } from "../../common/clients/storage/DataStoreStorageClient";
+import { StorageClient } from "../../common/clients/storage/StorageClient";
+import { Instance } from "../../common/entities/Instance";
+import { Namespaces } from "../../common/clients/storage/Namespaces";
+import { NamedRef, Ref } from "../../../domain/common/entities/Ref";
+import { D2Api, Pager } from "../../../types/d2-api";
+import {
+    AMCRecalculation,
+    ATCItem,
+    ATCItemIdentifier,
+    GLASSDataMaintenanceItem,
+    GLASSMaintenancePaginatedObjects,
+    GLASSModule,
+    Module,
+    Status,
+    getUserModules,
+} from "../../../domain/reports/glass-admin/entities/GLASSDataMaintenanceItem";
+import JSZip from "jszip";
+import _ from "lodash";
+import { Config } from "../../../domain/common/entities/Config";
+import { Id } from "../../../domain/common/entities/Base";
+import {
+    PaginatedObjects,
+    Paging,
+    Sorting,
+    getPaginatedObjects,
+} from "../../../domain/common/entities/PaginatedObjects";
+
+interface ATCJson {
```

Comment:
Try to avoid any in types. For this type you can take a look into glass repository at the end of [this file](https://github.com/EyeSeeTea/glass-dev/blob/38ef6e5c1fa16e614eb9855a7d24c067966fa877/src/data/repositories/GlassATCDefaultRepository.ts)